charts/redpanda/ci/20-certs-dns-names-values.yaml

```diff
@@ -0,0 +1,26 @@
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+tls:
+  enabled: true
+  certs:
+    default:
+      issuerRef:
+        name: custom-internal-issuer-ref
+        kind: ClusterIssuer
```

Comment:
The test will always fail as CI doesn't creates external issuer. It would require injecting `ClusterIssuer` in to template folder. That will only test `helm install`, but upgrade test would not work at all (`helm upgrade`) as `chart-testing` downloads previous version from the upstream.